lib/constants.js

```diff
@@ -6,7 +6,8 @@ module.exports = {
   sdkCodes: {
     analytics: 'AdobeAnalyticsSDK',
     target: 'AdobeTargetSDK',
-    campaign: 'CampaignSDK'
+    campaign: 'CampaignSDK',
+    customerProfile: 'CustomerProfileSDK'
```

Comment:
the official sdk code for all AEP APIs is `McDataServicesSdk` :)

Reply:
Could you rename the sdkcode `'CustomerProfileSDK'` to `'McDataServicesSdk'`
Thanks 🙂

Reply:
What about key, should I use just `aep` or `mcDataServices`?

Reply:
Updated